3.train_ebimage/1.random_forest/random-forest.py

```diff
@@ -0,0 +1,41 @@
+# %% Initialization
+import sys
+from pathlib import Path
+
+basefolder_loc = Path(__file__).parents[2]
+
+import os
+import numpy as np
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
+
+sys.path.append(str(basefolder_loc))
+from utils import create_metrics
+
+data_dir = os.path.join(basefolder_loc, "2.process-data", "data")
+
+
+def load_data(file_name="train_processed.tsv.gz"):
+    file_loc = os.path.join(data_dir, file_name)
+    data = pd.read_csv(file_loc, sep="\t")
+    X = np.array(data.drop(columns=["cell_code", "cell_id", "plate", "well", "target"]))
+    Y = np.array(data.target)
+    return X, Y
+
+
+# %% Training model
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import GridSearchCV
```

Comment:
Move all imports to the top of the file. All hail [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)!